src/test/resources/SampleJson/contentMigration/GetContentMigrationCompleted.json

```diff
@@ -2,10 +2,10 @@
 	"id": 44,
 	"user_id": 899123456,
 	"workflow_state": "completed",
-	"started_at": "2020-06-09T09:26:30Z",
-	"finished_at": "2020-06-09T09:26:52Z",
+	"started_at": "2020-06-09T09:26:30-06:00",
+	"finished_at": "2020-06-09T09:26:52-06:00",
 	"migration_type": "course_copy_importer",
-	"created_at": "2020-06-09T09:26:30Z",
+	"created_at": "2020-06-09T09:26:30-06:00",
```

Comment:
Was the original JSON not correct, or have Instructure changed how they are formatting the dates?

Reply:
I've seen it both ways (see some of the others json I've modified), but this new parser handles both formats, unlike the previous one.

Reply:
But shouldn't the JSON in the tests be in the format that Canvas returns as that's what we want to tests?

Reply:
Yes, that's why I changed it here. You can see it on:

https://canvas.instructure.com/doc/api/content_migrations.html

But also as I said I saw some dates with the Z format on some other Canvas responses/documentation.